post_installation_configuration/index.adoc

```diff
@@ -18,25 +18,20 @@ After installing {product-title}, a cluster administrator can configure and cust
 * Alerts and notifications
 
 [id="post-install-tasks"]
-== Configuration tasks to perform after installation
+== Post-installation configuration tasks
 
-Cluster administrators can perform the following postinstallation configuration tasks:
+You can perform the following post-installation configuration tasks:
```

Comment:
Optional update. SEO might truncate information so this incomplete sentence might reveal an incomplete sentence.

```suggestion
You can perform the post-installation configuration tasks to configure your environment to meet your need.

The following lists details these configurations:
```